components/browser/state/src/main/java/mozilla/components/browser/state/search/SearchEngine.kt

```diff
@@ -46,6 +46,11 @@ data class SearchEngine(
          * A custom search engine added by the user.
          */
         CUSTOM,
+
+        /**
+         * A special search engine add by the application.
+         */
+        SPECIAL,
```

Comment:
```suggestion
         * A custom search engine added by the application.
         */
        APPLICATION,
```

Maybe instead of special, we should just say application everywhere.

Reply:
Sure, agreed.  I'll update.